lua/orgmode/capture/init.lua

```diff
@@ -194,14 +194,19 @@ function Capture:_refile_content_with_fallback(lines, fallback_file, item)
   destination = vim.split(destination, '/', true)
 
   if not valid_destinations[destination[1]] then
+    if not default_file then -- we know that this comes from org_refile and not org_capture_refile
```

Comment:
@kristijanhusak this is confusing imo. `default_file` is nil when this function is not triggered from a capture buffer. These semantics are quite implicit, perhaps we can rethink how this works?

Reply:
It's nil in the situation when it's expected for a user to provide the destination file. I didn't handle the case where user didn't provide a valid destination. This is ok.